From oliver@neukum.org  Sat Sep  8 07:44:43 2007
From: Oliver Neukum <oliver@neukum.org>
Date: Tue, 4 Sep 2007 16:11:41 +0200
Subject: USB: quirky flash drive
To: gregkh@suse.de, linux-usb-devel@lists.sourceforge.net
Message-ID: <200709041611.42183.oliver@neukum.org>
Content-Disposition: inline


That drive is quite odd. It has 2K sectors, times out getting string
descriptors and needs a quirk.

Signed-off-by: Oliver Neukum <oneukum@suse.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/quirks.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/usb/core/quirks.c
+++ b/drivers/usb/core/quirks.c
@@ -28,6 +28,8 @@
  * devices is broken...
  */
 static const struct usb_device_id usb_quirk_list[] = {
+	/* CBM - Flash disk */
+	{ USB_DEVICE(0x0204, 0x6025), .driver_info = USB_QUIRK_RESET_RESUME },
 	/* HP 5300/5370C scanner */
 	{ USB_DEVICE(0x03f0, 0x0701), .driver_info = USB_QUIRK_STRING_FETCH_255 },
 	/* Hewlett-Packard PhotoSmart 720 / PhotoSmart 935 (storage) */
